base1: '1234567812345678'
samples: 20
number_of_antennas: 4
time_between_readings: 30